app/controllers/admin/auctions/needs_attention_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::Auctions::NeedsAttentionController < Admin::BaseController
+  def index
+    @view_model = Admin::NeedsAttentionAuctionsViewModel.new
```

Comment:
@jessieay should these be renamed, too? to something like `Admin::Auctions::NeedsAttentionViewModel`?